components/common-go/namegen/workspaceid.go

```diff
@@ -0,0 +1,465 @@
+// Copyright (c) 2020 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License-AGPL.txt in the project root for license information.
+
+package namegen
+
+import (
+	"math/rand"
```

Comment:
`uuid.NewRandom` (the one we used before) used `crypto/rand`. I suggest we used the same to make sure we have the same level of "randomness per bit".

Reply:
Impl seems straight forward:
```
var rander = rand.Reader // random function

func NewRandom() (UUID, error) {
	return NewRandomFromReader(rander)
}

// NewRandomFromReader returns a UUID based on bytes read from a given io.Reader.
func NewRandomFromReader(r io.Reader) (UUID, error) {
	var uuid UUID
	_, err := io.ReadFull(r, uuid[:])
	if err != nil {
		return Nil, err
	}
	uuid[6] = (uuid[6] & 0x0f) | 0x40 // Version 4
	uuid[8] = (uuid[8] & 0x3f) | 0x80 // Variant is 10
	return uuid, nil
}
```
(from [here](https://github.com/google/uuid/blob/bfb86fa49a73e4194d93bea18d7acfe3694438ce/version4.go#L37))